aireview/src/aireview/config.py

```diff
@@ -1,11 +1,11 @@
 from pydantic_settings import BaseSettings
 
-class Config(BaseSettings):
+class AIReviewSettings(BaseSettings):
     dust_api_key: str
     github_token: str
```

Comment:
**documentation**: Consider adding docstring documentation to describe the purpose of the configuration class and its fields.